README.md

```diff
@@ -102,7 +109,7 @@ use the real AWS account ID.
 Example:
 
 ```shell
-go run cmd/main.go --role engineer --iam-user testuser --account-id 123456789012  --profile test-profile-name
+go run cmd/main.go setup --iam-role engineer --iam-user testuser --aws-profile test-profile-name --aws-account-id 123456789012
```

Comment:
I assume it's just my set up... this line no longer works in general... the make bin/setup-new-aws-user works though... and the binaryit builds works...

```➜  setup-new-aws-user git:(cg_new_cli_lib) go run cmd/main.go setup --iam-role engineer --iam-user testuser --aws-profile test-profile-name --aws-account-id 123456789012
# command-line-arguments
cmd/main.go:31:26: undefined: setupUserFunction
cmd/main.go:33:2: undefined: setupUserInitFlags
cmd/main.go:41:26: undefined: versionFunction
➜  setup-new-aws-user git:(cg_new_cli_lib) ./bin/setup-new-aws-user setup --iam-role engineer --iam-user testuser --aws-profile test-profile-name --aws-account-id 123456789012
Checking whether profile "test-profile-name-base" exists in AWS config file
Checking whether profile "test-profile-name" exists in AWS config file
Enter Access Key ID: ^C```

Reply:
ahh, I forgot to modify this to be `go run ./cmd`. One sec.